### PR TITLE
Check for upgrades

### PR DESCRIPTION
**Pull request by crosbymichael**
Signed-off-by: Michael Crosby <crosbymichael@gmail.com>

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
 )
 
-type Config struct {
-	ID        string   `json:"-"`
-	Namespace string   `json:"namespace"`
-	Image     string   `json:"image"`
-	ImagePath string   `json:"imagePath"`
-	Args      []string `json:"args"`
-}
+const (
```
Comment (seemethere) [inline config.go]: Can we add a comment here that these are meant to be overridable at build time?

**Review by seemethere — COMMENTED** [review]
just one comment about docs

File: main.go [hunk 2]
```diff
@@ -58,8 +46,15 @@ func proxy(ctx context.Context, config *Config, signals chan os.Signal) error {
...
 			return err
 		}
+		image, err := config.GetImage(ctx, client)
+		if err != nil {
+			return err
+		}
 		// create new container
-		if container, err = create(ctx, client, config); err != nil {
+		if container, err = client.NewContainer(ctx, config.ID,
+			WithCurrentSpec(config),
```
Comment (seemethere) [inline main.go:55]: Does `Scope` automatically get handled here?

Comment (crosbymichael) [reply main.go:55]: no, i'm not sure we need anything specific to scope

Comment (seemethere) [reply main.go:55]: Or sorry, I meant does this handle setting the label for scope?

Comment (seemethere) [reply main.go:55]: Since we'll need to set initial scope on container creation right?

Comment (crosbymichael) [reply main.go:55]: ahh, let me add that

Comment (crosbymichael) [reply main.go:55]: fixed

**Review by seemethere — APPROVED** [review]
LGTM

**Pull request merged**